src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -0,0 +1,106 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.HttpMethod;
+import javax.ws.rs.core.MediaType;
+
+import org.kuali.kfs.krad.service.BusinessObjectService;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientRequest;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.config.ClientConfig;
+import com.sun.jersey.api.client.config.DefaultClientConfig;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.ConcurPropertyConstants;
+import edu.cornell.kfs.concur.rest.xmlObjects.AccessTokenDTO;
+import edu.cornell.kfs.concur.service.ConcurAccessTokenService;
+import edu.cornell.kfs.sys.businessobject.WebServiceCredential;
+import edu.cornell.kfs.sys.service.WebServiceCredentialService;
+
+public class ConcurAccessTokenServiceImpl implements ConcurAccessTokenService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurAccessTokenServiceImpl.class);
+
+    protected String concurRefreshAccessTokenURL;
```

Comment:
Could this string variable be private even if the service is protected (which we normally do because of possible overriding)?